packages/ButtonGroup/ButtonGroupItem/ButtonGroupItem.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import safeRest from '../../../shared/utils/safeRest'
+
+import styles from './ButtonGroupItem.modules.scss'
+
+const ButtonGroupItem = ({
+  name,
+  value,
+  selectedValue,
+  onChange,
+  onFocus,
+  onBlur,
+  children,
+  ...rest
+}) => (
+  <div className={styles.itemContainer} {...safeRest(rest)}>
+    <input
+      id={value}
+      name={name}
+      value={value}
+      type="radio"
+      checked={selectedValue === value}
```

Comment:
Not entirely sure what you mean here. Currently, the ButtonGroup does control the selected value through its `value` prop.

Reply:
I'm thinking about ButtonGroup.Item.

Instead of passing it `selectedValue` and `value` props, can we just pass it a `checked` prop?

Something like this:
```
const getButtons = () => {
    return React.Children.map(children, child =>
      React.cloneElement(child, { name, onChange, onFocus, onBlur, checked: value === child.props.value })
    )
  }
```

Reply:
Oh, I see. Sure, I'm good with that.